packages/react-server/core/__tests__/integration/redirectForward/RedirectForwardSpec.js

```diff
@@ -1,12 +1,12 @@
-var helper = require("../specRuntime/testHelper"),
+var helper = require("../../../test/specRuntime/testHelper"),
```

Comment:
Why don't we want `react-server-test-helpers` in npm?  Could they be useful for users to test their pages?